recipes/xtl/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "xtl" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "97cb01feb042ee59727ea1dcf502ed300c3533f009a6e606094b2811c66ee1f7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantStack/xtl/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/xtl  # [unix]
+    - test -f ${PREFIX}/include/xtl/xbasic_fixed_string.hpp  # [unix]
+    - test -f ${PREFIX}/lib/cmake/xtl/xtlConfig.cmake  # [unix]
+    - test -f ${PREFIX}/lib/cmake/xtl/xtlConfigVersion.cmake  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\xtl\xbasic_fixed_string.hpp (exit 0) else (exit 1)  # [win]
```

Comment:
Totally don't need to change this, but FYI `%LIBRARY_PREFIX%\include` is more shortly written as `%LIBRARY_INC%`, and similarly `%LIBRARY_LIB%` for `lib` (https://conda.io/docs/user-guide/tasks/build-packages/environment-variables.html#environment-variables-set-during-the-build-process).

Reply:
Thx for the info, I'll remember that for my next recipes.